src/pybind/mgr/nfs/module.py

```diff
@@ -183,8 +183,10 @@ def _cmd_nfs_cluster_config_reset(self, cluster_id: str) -> None:
     def fetch_nfs_export_obj(self) -> ExportMgr:
         return self.export_mgr
 
-    def export_ls(self) -> List[Dict[Any, Any]]:
-        return self.export_mgr.list_all_exports()
+    def export_ls(self, cluster_id:str=None, detailed=False) -> List[Dict[Any, Any]]:
+        if not(cluster_id):
+            return self.export_mgr.list_all_exports()
+        return self.export_mgr.list_exports(cluster_id,detailed)
```

Comment:
okay, i see now what you are doing in the above code as well. looks good. Since these are maintained by other folks probably we need their attention as well. @adk3798 @phlogistonjohn please have a look.